<launch>

  <node pkg = "rospkg-ogamesure"
        name = "fsw0020"
        type = "fsw0020.py">
        <param name="~host" value="" />
        <param name="~port" value="" />
    <remap from = '/dev/fsw0020/__IP__/freq_cmd' to = ''/>
    <remap from = '/dev/fsw0020/__IP__/power_cmd' to = ''/>
    <remap from = '/dev/fsw0020/__IP__/onoff_cmd' to = ''/>
    <remap from = '/dev/fsw0020/__IP__/freq' to = ''/>
    <remap from = '/dev/fsw0020/__IP__/power' to = ''/>
    <remap from = '/dev/fsw0020/__IP__/onoff' to = ''/>

  </node>

</launch>
